modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/exp/ExpressionFactoryImpl.java

```diff
@@ -105,13 +105,13 @@ public class ExpressionFactoryImpl<Row> implements ExpressionFactory<Row> {
     private final RexBuilder rexBuilder;
 
     /** */
-    private final RelDataType emptyType;
+    private static final RelDataType emptyType = new RelDataTypeFactory.Builder(Commons.typeFactory()).build();
```

Comment:
Usually we use upper case for `static final` constants. Same below

Reply:
Fixed